moveit_core/trajectory_processing/test/test_time_optimal_trajectory_generation.cpp

```diff
@@ -274,96 +306,14 @@ TEST(time_optimal_trajectory_generation, testLastWaypoint)
-  add_waypoint({ 0.011108776, 0.011108776 });
-  add_waypoint({ 0.011505518, 0.011505518 });
-  add_waypoint({ 0.011902261, 0.011902261 });
-  add_waypoint({ 0.012299003, 0.012299003 });
-  add_waypoint({ 0.012695745, 0.012695745 });
-  add_waypoint({ 0.013092487, 0.013092487 });
-  add_waypoint({ 0.013489229, 0.013489229 });
-  add_waypoint({ 0.013885971, 0.013885971 });
-  add_waypoint({ 0.014282713, 0.014282713 });
-  add_waypoint({ 0.014679455, 0.014679455 });
-  add_waypoint({ 0.015076197, 0.015076197 });
-  add_waypoint({ 0.015472939, 0.015472939 });
-  add_waypoint({ 0.015869681, 0.015869681 });
-  add_waypoint({ 0.016266423, 0.016266423 });
-  add_waypoint({ 0.016663165, 0.016663165 });
-  add_waypoint({ 0.017059907, 0.017059907 });
-  add_waypoint({ 0.017456649, 0.017456649 });
-  add_waypoint({ 0.017853391, 0.017853391 });
-  add_waypoint({ 0.018250133, 0.018250133 });
-  add_waypoint({ 0.018646875, 0.018646875 });
-  add_waypoint({ 0.019043617, 0.019043617 });
-  add_waypoint({ 0.019440359, 0.019440359 });
-  add_waypoint({ 0.019837101, 0.019837101 });
-  add_waypoint({ 0.020233843, 0.020233843 });
-  add_waypoint({ 0.020630585, 0.020630585 });
-  add_waypoint({ 0.021027327, 0.021027327 });
-  add_waypoint({ 0.021424069, 0.021424069 });
-  add_waypoint({ 0.021820811, 0.021820811 });
-  add_waypoint({ 0.022217553, 0.022217553 });
-  add_waypoint({ 0.022614295, 0.022614295 });
-  add_waypoint({ 0.023011037, 0.023011037 });
-  add_waypoint({ 0.023407779, 0.023407779 });
-  add_waypoint({ 0.023804521, 0.023804521 });
-  add_waypoint({ 0.024201263, 0.024201263 });
-  add_waypoint({ 0.024598005, 0.024598005 });
-  add_waypoint({ 0.024994747, 0.024994747 });
-  add_waypoint({ 0.025391489, 0.025391489 });
-  add_waypoint({ 0.025788231, 0.025788231 });
-  add_waypoint({ 0.026184973, 0.026184973 });
-  add_waypoint({ 0.026581715, 0.026581715 });
-  add_waypoint({ 0.026978457, 0.026978457 });
-  add_waypoint({ 0.027375199, 0.027375199 });
-  add_waypoint({ 0.027771941, 0.027771941 });
-  add_waypoint({ 0.028168683, 0.028168683 });
-  add_waypoint({ 0.028565425, 0.028565425 });
-  add_waypoint({ 0.028962167, 0.028962167 });
-  add_waypoint({ 0.029358909, 0.029358909 });
-  add_waypoint({ 0.029755651, 0.029755651 });
-  add_waypoint({ 0.030152393, 0.030152393 });
-  add_waypoint({ 0.030549135, 0.030549135 });
-  add_waypoint({ 0.030945877, 0.030945877 });
-  add_waypoint({ 0.031342619, 0.031342619 });
-  add_waypoint({ 0.031739361, 0.031739361 });
-  add_waypoint({ 0.032136103, 0.032136103 });
-  add_waypoint({ 0.032532845, 0.032532845 });
-  add_waypoint({ 0.032929587, 0.032929587 });
-  add_waypoint({ 0.033326329, 0.033326329 });
-  add_waypoint({ 0.033723071, 0.033723071 });
-  add_waypoint({ 0.034119813, 0.034119813 });
-  add_waypoint({ 0.034516555, 0.034516555 });
-
-  const std::vector<double> expected_last_waypoint = { 0.034913297, 0.034913297 };
+  add_waypoint({ 0.000000000, 0.000000000, 0, 0, 0, 0, 0 });
```

Comment:
I guess you removed a number of waypoints because the test trajectory does not have to be that long, right?

Reply:
yeah, exactly.